frontend/src/index.jade

```diff
@@ -75,36 +75,42 @@ html
               input(type="checkbox", v-model="filterGroupRepos").summary-picker__checkbox
               span group by repo
         #summary-charts
-          .summary-charts(v-for="repo of filtered", v-if="repo.length>0")
-            a.summary-charts__title(
-              v-if="filterGroupRepos",
-              v-bind:href="repo[0].location", target="_blank"
-            ) {{ repo[0].repoPath }}
-            .summary-chart(v-for="(user, i) in repo", v-bind:class="{indented:filterGroupRepos}")
-              .summary-chart__title(
-                v-on:click="$emit('view-authorship', {author:user.name, repo:user.repoName, name:user.displayName})"
+          .summary-charts(v-for="(repo, repoId) of filtered", v-bind:key="repoId")
```

Comment:
`of` and `in` does the same thing right? in that case, we should stick to a single standard